src/lib/types/IndividualCoin.ts

```diff
@@ -0,0 +1,345 @@
+  watchlist_portfolio_users: number;
+  market_cap_rank: number;
+  market_data: Marketdata;
+  community_data: Communitydata;
+  developer_data: Developerdata;
+  status_updates: any[] | null;
+  last_updated: string;
+}
+
+export interface IndividualCoinWith24hVolume extends IndividualCoin {
+  volume_24h: number | null;
+}
+
+interface Developerdata {
+  forks: number;
+  stars: number;
+  subscribers: number;
+  total_issues: number;
+  closed_issues: number;
+  pull_requests_merged: number;
+  pull_request_contributors: number;
+  code_additions_deletions_4_weeks: Codeadditionsdeletions4weeks;
+  commit_count_4_weeks: number;
+  last_4_weeks_commit_activity_series: any[] | null;
+}
+
+interface Codeadditionsdeletions4weeks {
+  additions: number;
+  deletions: number;
+}
+
+interface Communitydata {
+  facebook_likes: any | null;
+  twitter_followers: number;
+  reddit_average_posts_48h: number;
+  reddit_average_comments_48h: number;
+  reddit_subscribers: number;
+  reddit_accounts_active_48h: number;
+  telegram_channel_user_count: any | null;
+}
+
+interface Marketdata {
+  current_price: Currentprice | {};
+  total_value_locked: any | null;
+  mcap_to_tvl_ratio: any | null;
+  fdv_to_tvl_ratio: any | null;
+  roi: any | null;
+  ath: Currentprice | {};
+  ath_change_percentage: Currentprice;
+  ath_date: Athdate | {};
+  atl: Currentprice | {};
+  atl_change_percentage: Currentprice;
+  atl_date: Athdate | {};
+  market_cap: Currentprice | {};
+  market_cap_rank: number | null;
+  fully_diluted_valuation: Currentprice | {};
+  market_cap_fdv_ratio: number | null;
+  total_volume: Currentprice | {};
+  high_24h: Currentprice | EmptyHighLow;
+  low_24h: Currentprice | EmptyHighLow;
+  price_change_24h: number | null;
+  price_change_percentage_24h: number | null;
+  price_change_percentage_7d: number;
+  price_change_percentage_14d: number;
+  price_change_percentage_30d: number;
+  price_change_percentage_60d: number;
+  price_change_percentage_200d: number;
+  price_change_percentage_1y: number;
+  market_cap_change_24h: number | null;
+  market_cap_change_percentage_24h: number | null;
+  price_change_24h_in_currency: Currentprice | {};
+  price_change_percentage_1h_in_currency: Currentprice | {};
+  price_change_percentage_24h_in_currency: Currentprice | {};
+  price_change_percentage_7d_in_currency: Currentprice | {};
+  price_change_percentage_14d_in_currency: Currentprice | {};
+  price_change_percentage_30d_in_currency: Currentprice | {};
+  price_change_percentage_60d_in_currency: Currentprice | {};
+  price_change_percentage_200d_in_currency: Currentprice | {};
+  price_change_percentage_1y_in_currency: Currentprice | {};
+  market_cap_change_24h_in_currency: Currentprice | {};
+  market_cap_change_percentage_24h_in_currency: Currentprice | {};
+  total_supply: number | null;
+  max_supply: number | null;
+  circulating_supply: number | null;
+  last_updated: string | null;
+}
+
+interface Athdate {
```

Comment:
interface Athdate { 
[key: string]: string
}

https://stackoverflow.com/questions/39256682/how-to-define-an-interface-for-objects-with-dynamic-keys